src/plugins/intel_cpu/tests/functional/subgraph_tests/src/subgraph_serialize.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/openvino.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "test_utils/convolution_params.hpp"
+#include "snippets/op/subgraph.hpp"
+
+using namespace CPUTestUtils;
+
+namespace SubgraphTestsDefinitions {
+
+class SubgraphSnippetSerializationTest : public ::testing::Test, public CPUTestsBase {};
+
+TEST_F(SubgraphSnippetSerializationTest, SerializeSubgraph) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    auto model = ([] () -> std::shared_ptr<ov::Model> {
+        auto shape = ov::Shape({2, 2});
+        auto input0 = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, shape);
```

Comment:
Detail: Consider add:
#include "openvino/opsets/opset9.hpp"
using namespace ov::opset9;

Then it should be possible to change all ops creation to and can easy switch opset for all of them.
```suggestion
        auto input0 = std::make_shared<Parameter>(ov::element::f32, shape);
```